<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1031</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.652, y2:0.897727, stop:0.243781 rgba(83, 187, 255, 255), stop:1 rgba(255, 179, 249, 255))};</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>461</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:15px;

background-color: rgb(255, 255, 255); </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;MS Sans Serif&quot;;
color:rgb(45, 169, 203)</string>
    </property>
    <property name="text">
     <string> Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_user_name_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;

background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> USER NAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_password_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

 font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_confirmedpassword_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> CONFIRM PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_emailid_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> EMAIL ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_mobileno_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> MOBILE NO.</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="reg_resaddress_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>371</width>
      <height>87</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> RESIDENTIAL ADDRESS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reg_pincode_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Border-radius:10px;

font: 75 12pt &quot;MS Sans Serif&quot;;
background-color: rgb(229, 250, 255);</string>
    </property>
    <property name="placeholderText">
     <string> PINCODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>690</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
background-color: rgb(4, 118, 159)
}

QPushButton:hover {
  background-color:rgb(0, 0, 0)
};

color:rgb(255, 255, 255);
Border-radius:10px;

font: 75 15pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>461</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/newPrefix/download.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/newPrefix/download.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 149, 245);

font: 22pt &quot;MS Sans Serif&quot;;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>                           Health Management App</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_error_msg_text_field">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>630</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_signin_button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>690</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
background-color: rgb(4, 118, 159)
}

QPushButton:hover {
  background-color:rgb(0, 0, 0)
};

color:rgb(255, 255, 255);
Border-radius:10px;

font: 75 15pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="registration_img.qrc"/>
 </resources>
 <connections/>
</ui>
